tests/rptest/services/redpanda.py

```diff
@@ -2183,6 +2183,11 @@ def write_bootstrap_cluster_config(self):
                     self._extra_rp_conf))
             conf.update(self._extra_rp_conf)
 
+        if cur_ver != RedpandaInstaller.HEAD and cur_ver < (22, 2, 1):
+            # this configuration property was introduced in 22.2.1, ensure
+            # it doesn't appear in older configurations
+            conf.pop('cloud_storage_credentials_source', None)
```

Comment:
should this go into the tests that are setting this property since ostensibly they are aware of the versions they are using too?

Reply:
AIUI the property is coming from the overall test setup, where we set this property if we see that we're running on EC2 nodes.

Reply:
ohh, interesting. thanks for the context